src/test/utils/VaultUtils.sol

```diff
@@ -271,6 +313,43 @@ abstract contract BasicVaultTest is UniswapV2Fixture {
         assertEq(getToken0Active(), 0);
     }
 
+    function test_advanceWithNoUserDepositedTokens() public {
+        depositToken0();
```

Comment:
is this required because the vault fails to rollover if there are no deposits at all, instead of failing with `NOT_PARTICIPANT_OR_STRATEGIST`?

Reply:
It seems like this isn't necessary, as the revert will happen before any vault logic takes place. Removing.